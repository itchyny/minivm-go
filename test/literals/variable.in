foo = 10
bar = 20 + 3 * foo
print foo
print bar
print 3 * foo + 4 * bar
