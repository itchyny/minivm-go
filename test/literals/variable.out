10
50
230
